src/lib/third_party/include/ndpi_crc.h

```diff
@@ -0,0 +1,8 @@
+#ifndef _NDPI_CRC_
+#define _NDPI_CRC_
+
+u_int16_t ndpi_crc16_ccit(const void* data, size_t n_bytes);
+u_int16_t ndpi_crc16_ccit_false(const void *data, size_t n_bytes);
+u_int16_t ndpi_crc16_xmodem(const void *data, size_t n_bytes);
+
+#endif
```

Comment:
Could you avoid adding a new header and move these declaration in `src/include/ndpi_api.h`, near the similar crc32 ones, please?